tests/integration/goldens/logging/samples/generated_samples/logging_generated_logging_v2_logging_service_v2_tail_log_entries_async.py

```diff
@@ -38,8 +38,17 @@ async def sample_tail_log_entries():
         resource_names=['resource_names_value_1', 'resource_names_value_2'],
     )
 
+    # This method expects an iterator which contains
+    # 'logging_v2.TailLogEntriesRequest' objects
+    # Here we create a generator that yields a single `request` for
+    # demonstrative purposes.
+    requests = [request]
```

Comment:
One alternative is:

```py
requests = iter([requests])
```
I chose the generator over `iter()` to nudge folks towards generating requests on demand.

Opinions/suggestions on the best way to present client streaming welcome!